documentation/library_guide/pstl/dpcpp_policies_usage.rst

```diff
@@ -175,11 +175,22 @@ allow you to pass SYCL buffers to parallel algorithms. These functions accept
 a SYCL buffer and return an object of an unspecified type that satisfies the following
 requirements:
 
-* Is ``CopyConstructible``, ``CopyAssignable``, and comparable with operators == and !=
-* The following expressions are valid: ``a + n``, ``a - n``, and ``a - b``, where ``a`` and ``b``
+* it is ``CopyConstructible``, ``CopyAssignable``, and comparable with operators ``==`` and ``!=``;
+* the following expressions are valid: ``a + n``, ``a - n``, and ``a - b``, where ``a`` and ``b``
```

Comment:
You are saying what expressions are valid but you are not saying the effect from those

Reply:
You are right. Do you think it needs to be mentioned? What if we say that "it has the same effect as the return type would be an iterator"?

Reply:
I think it would be ok if you say something like "effect for those operations is the same as for the type that satisfies ``LegacyRandomAccessIterator`` C++ named requirement." It might be still unclear, though. Probably better wording is needed.